zastosowania-komputerow-w-elektrotechnice.csl

```diff
@@ -0,0 +1,326 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Zastosowania Komputerów w Elektrotechnice</title>
+    <title-short>ZKwE</title-short>
+    <id>http://www.zotero.org/styles/zastosowania-komputerow-w-elektrotechnice</id>
+    <link href="http://www.zotero.org/styles/zastosowania-komputerow-w-elektrotechnice" rel="self"/>
+    <link href="http://www.zotero.org/styles/ieee" rel="template"/>
+    <link href="http://www.iee.put.poznan.pl/caiee/redakcja.doc" rel="documentation"/>
+    <author>
+      <name>Mateusz Pacek</name>
+      <email>mateusz.m.pacek@student.put.poznan.pl</email>
+    </author>
+    <contributor>
+      <name>Michael Berkowitz</name>
+      <email>mberkowi@gmu.edu</email>
+    </contributor>
+    <category citation-format="numeric"/>
+    <category field="engineering"/>
+    <category field="generic-base"/>
```

Comment:
could you remove this line -- the style isn't "generic" -- it's an engineering style.